rust_dev_preview/dynamodb/src/bin/add-item.rs

```diff
@@ -3,12 +3,28 @@
  * SPDX-License-Identifier: Apache-2.0.
  */
 
-use aws_config::meta::region::RegionProviderChain;
-use aws_sdk_dynamodb::model::AttributeValue;
-use aws_sdk_dynamodb::{Client, Error, Region, PKG_VERSION};
-use std::process;
+use std::fmt::Display;
+
+use aws_sdk_dynamodb::{Client, Error};
+use dynamodb_code_examples::{
+    make_config,
+    scenario::create::{add_item, Item},
+    Opt as BaseOpt,
+};
 use structopt::StructOpt;
 
+#[derive(Debug)]
+struct PermissionError {
+    p_type: String,
+}
+
+impl std::error::Error for PermissionError {}
+impl Display for PermissionError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
```

Comment:
This is merely bikeshedding, but importing `std::fmt` seems like the nicest way to do this to me:
```suggestion
impl fmt::Display for PermissionError {
    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
```